### DISCUSSION TITLE
Add Controls for Keyboard Backlight?

### DISCUSSION THREAD

**Discussion by isametry** [MonitorControl/MonitorControl#886] [Ideas]
I think it would be really useful to have the option for a keyboard backlight brightness slider in this app.

Now, I know that it might seem a little off-topic for this app, since it is focused on monitors (and literally has the word _"monitor"_ in the name), but it's one of the native brightness sliders provided by macOS (in the Control Center), and having it alongside the monitor ones would simply be very practical.

Alternatively, if anyone knows about a tool as simple and native-looking as _MonitorControl_ which allows this, please let me know. Thank you!

**Comment by waydabber** [comment]
Technically it could be done and might make sense.

**Reply by waydabber** [reply]
https://github.com/MonitorControl/MonitorControl/issues/887
